pages/technical-and-interface-writing.md

```diff
@@ -0,0 +1,131 @@
+---
+permalink: /technical-and-interface-writing/
+title: Technical and interface writing
+---
+
+At 18F, we often write technical documentation, guides, forms, and interface messages. In most of these cases, it’s safe to say the reader is learning something new or troubleshooting. These guidelines will help you write clear, concise instructions, which will provide your reader with the best possible experience.
+
+## Basics
+
+### Do the hard work to make it simple.
+
+Help the reader follow along. Break instructions or processes down into individual steps. Use short, simple sentences with words people use in everyday conversation.
+
+Refer to navigation labels, buttons, and menus as they appear in the app or website. Verify the spelling and capitalization as you write. Be specific.
+
+Instead of:
+
+> Open a new meeting invitation.
+
+Try:
+
+> In Google Calendar, click **Create**.
```

Comment:
I did go ahead and add a bit lower down where we're talking about interface directions.